packages/graphback-datasync/src/helpers/createDataSync.ts

```diff
@@ -0,0 +1,23 @@
+import { GraphQLSchema } from 'graphql';
+import { Db } from 'mongodb';
+import { GraphbackAPIConfig, GraphbackAPI, buildGraphbackAPI, GraphbackConfig } from "graphback";
+import { PubSub } from 'graphql-subscriptions';
+import { createDataSyncMongoDbProvider, createDataSyncConflictProviderCreator } from '../providers';
+import { ConflictError, ConflictMetadata, DataSyncModelConflictConfig } from "../util";
+import { createDataSyncCRUDService } from '../services';
+import { DataSyncPlugin } from '../DataSyncPlugin';
+
+type DataSyncGraphbackConfig = Partial<GraphbackConfig>
```

Comment:
`GraphbackConfig` is not used in runtime API (it is used for the `generate` command. I think you are looking for `GraphbackAPIConfig`?